### PR TITLE
Issue #237: Refactor `HelpWindow`

### LINKED ISSUE #237: Update UI `HelpWindow`
Some minor refactors.

### PR DESCRIPTION
**Pull request by domlimm**
This PR consists of the following changes:
- Update user guide link to link to command summary
- Refactor position for button in window
- Update help window header

Closes #237.

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -16,8 +16,8 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL =
-            "https://ay2122s2-cs2103-f11-2.github.io/tp/UserGuide.html";
-    private static final String HELP_MESSAGE = "Available functions on this version: \n\n";
+            "https://ay2122s2-cs2103-f11-2.github.io/tp/UserGuide.html#command-summary";
```
Comment (tiewweijian) [inline src/main/java/seedu/address/ui/HelpWindow.java:19]: Do we want to link them directly to the command summary ah? Cause what I intended was just to bring them to the full guide hahaha ! Up to you ah !

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
+    private static final String HELP_MESSAGE = "Available commands on this version";
```
Comment (tiewweijian) [inline src/main/java/seedu/address/ui/HelpWindow.java:20]: 👍🏻

**Review by tiewweijian — APPROVED** [review]
LGTM ! Just a small thing regarding the URL!

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (domlimm) [reply src/main/java/seedu/address/ui/HelpWindow.java:19]: Good point! I'll revert it back. Realised it's `Open User Guide in browser`.

File: src/main/resources/view/HelpWindow.fxml [hunk 3]
```diff
@@ -58,15 +58,15 @@
                 <Label fx:id="exit" styleClass="styleCommand" text="exit" prefWidth="70" alignment="CENTER" />
             </HBox>
         </children>
-        <HBox styleClass="styleHBox" alignment="BOTTOM_LEFT">
+        <HBox fx:id="copyButtonHBox" alignment="BASELINE_RIGHT">
```
Comment (LeeEnEn) [inline src/main/resources/view/HelpWindow.fxml:61]: Nice adjustment to the right!

**Review by LeeEnEn — APPROVED** [review]
Lgtm!

**Pull request merged**